coreax/util.py

```diff
@@ -281,6 +282,7 @@ def jit_test(
     fn_args: tuple = (),
     fn_kwargs: Optional[dict] = None,
     jit_kwargs: Optional[dict] = None,
+    _name: Optional[str] = None,  # UNUSED
```

Comment:
Rather than noting this as unused, could we just do `del _name` and include a comment explaining why this parameter is necessary here